main.tex

```diff
@@ -976,63 +986,120 @@ \subsection{Convolution}
 \subsection{Visualization}
 % Larry:  Image visualization (stretching, scaling), RGB
 
-The \package{astropy.visualization} subpackage provides functionality that can be helpful when visualizing data. This includes a framework for plotting astronomical images with coordinates with \package{matplotlib} (previously the standalone \package{wcsaxes} package), functionality related to image normalization (including both scaling and stretching), smart histogram plotting, RGB color image creation from separate images, and custom plotting styles for \package{matplotlib}.
+The \package{astropy.visualization} subpackage provides functionality that can
+be helpful when visualizing data. This includes a framework (previously the
```

Comment:
Note: I moved the parenthesis up a little to prevent readers from wrongly thinking that it is linked to `matplotlib`.